snippets/assertions.json

```diff
@@ -0,0 +1,72 @@
+{
+	"tpass": {
+		"prefix": "tpass",
```

Comment:
The key of the object isn't the prefix. It's just a unique key and is called [the snippet name](https://code.visualstudio.com/Docs/customization/userdefinedsnippets#_creating-your-own-snippets) :).